frontend/.eslintrc.json

```diff
@@ -10,7 +10,7 @@
       ],
       "parserOptions": {
         "project": [
-          "/workspace/frontend/tsconfig.json"
+          "./tsconfig.json"
```

Comment:
This change breaks `eslint`; please validate the PR does not break anything before resubmitting. To validate, open a TS file in `frontend` (such as `src/main.ts`) and ensure that you do not see an error such as `Cannot find file: /workspace/tsconfig.json`. In this case, the editor/eslint is looking relative to project root not frontend. Please fix and push again.

Reply:
I have. It works. `./frontend/tsconfig.json` breaks it. ESLint is searching from relative to where `.eslintrc.json` is located not the project root.

Reply:
This was tested in the devcontainer and breaks the devcontainer.

Reply:
I'll change it to ./frontend/tsconfig.json for now then and investigate further to try and find a way to get this to work everywhere.